packages/framework/src/Command/DatabaseDumpCommand.php

```diff
@@ -11,7 +11,9 @@
 
 class DatabaseDumpCommand extends Command
 {
+    /** @access private */
     const ARG_OUTPUT_FILE = 'outputFile';
```

Comment:
Shouldn't argument names be public? I'd guess they might come handy when calling the command programatically or during an extension. As an example, take a look at `ServerStartWithCustomRouterCommand` which would definitely use a `ServerStartCommand::ARG_ROUTER` constant if it were declared (now it uses only the literal string `'router'`).

Or did you have some other reason for the private visibility in mind?

Reply:
Well, this seems to me like an edge case because they're inherited classes. I don't think it's commonly needed to inherit commands.
I don't want to use argument and option names anywhere else, because I don't see it as a need and don't know why I should need it (except extending command).

That's why I choose private :)